src/store/ProductContext.js

```diff
@@ -59,7 +59,41 @@ export const ProductsContextProvider=({children})=>{
                         ...state,
                         wishListItems:state.wishListItems.filter(product=>product.id!==action.payload),
                         products:state.products.map(product=>product.id===action.payload?{...product,inWishlist:false}:product)
-                     }
+                     };
+            case "INCREMENT_QUANTITY":
+                      return{
+                          ...state,
+                          cartItems:state.cartItems.map(product=>product.id===action.payload?{...product,quantity:product.quantity+1}:product)
+                      };
+            case "DECREMENT_QUANTITY":
+                      return{
+                          ...state,
+                          cartItems:state.cartItems.map(product=>product.id===action.payload?{...product,quantity:product.quantity>1?product.quantity-1:product.quantity}:product)
+                      };
+            
+            case "ADD_TO_CART":
+      
+                   return{
+                      ...state,
+                        cartItems:[...state.cartItems,...state.products.filter(product=>product.id===action.payload).map(item=>({...item,inCart:true,quantity:1}))],
```

Comment:
could you give appropriate spacing between the keys, and spacing between the object's values.